Sources/Extensions/UIWindow+RootViewController.swift

```diff
@@ -21,15 +21,10 @@ public extension UIWindow {
      - parameter completion:     Completion block to be invoked after the transition finishes
      */
     @nonobjc
-    func setRootViewController(_ viewController: UIViewController, animated: Bool, duration: TimeInterval = 0.3, completion: ((Bool) -> Void)? = nil) {
+    func setRootViewController(_ viewController: UIViewController, animated: Bool, duration: TimeInterval = 0.3, options: UIView.AnimationOptions = [.transitionCrossDissolve], completion: ((Bool) -> Void)? = nil) {
```

Comment:
Brackets are optional, since the `UIView.AnimationOptions` is an `OptionSet`.

```suggestion
    func setRootViewController(_ viewController: UIViewController, animated: Bool, duration: TimeInterval = 0.3, options: UIView.AnimationOptions = .transitionCrossDissolve, completion: ((Bool) -> Void)? = nil) {
```